cms/templates/ux/reference/modal_access-component.html

```diff
@@ -0,0 +1,126 @@
+<div class="wrapper wrapper-modal-window wrapper-modal-window-edit-xblock" aria-describedby="modal-window-description" aria-labelledby="modal-window-title" aria-hidden="" role="dialog">
+  <div class="modal-window-overlay"></div>
+  <div class="modal-window confirm modal-med modal-type-html modal-editor" style="top: 50px; left: 400px;">
+    <div class="edit-xblock-modal">
+      <div class="modal-header">
+        <h2 class="title modal-window-title">Editing visibility for: [Component Name]</h2>
+      </div>
+
+      <div class="modal-content">
+        <div class="xblock-editor" data-locator="i4x://TestU/cohorts001/chapter/748152225449412a846bc24811a5621c" data-course-key="">
+
+          <div class="xblock xblock-visibility_view">
+
+            <div class="modal-section visibility-summary">
+              <div class="summary-message summary-message-warning visibility-summary-message">
+                <i class="icon icon-warning-sign"></i>
+                <p class="copy">This component is contained in a unit that is hidden from students. Component visibility settings are overridden by the unit visibility settings.</p>
+              </div>
+
+              <!-- NOTE: use when no group configuration has been set -->
+              <div class="is-not-configured has-actions">
+                <h4 class="title">You have not set up any groups</h4>
+
+                <div class="copy">
+                  <p>Groups are a way for you to organize content in your course with a particular student experience in mind. They are commonly used to facilitate content and pedagogical experiments as well as to provide different tracks of content.</p>
+                </div>
+
+                <div class="actions">
+                  <a href="" class="action action-primary action-settings">Manage groups in this course</a>
+                </div>
+              </div>
+            </div>
+
+            <form class="visibility-controls-form" method="post" action="">
+
+              <div class="modal-section visibility-controls">
+                <h3 class="modal-section-title">Set visibility to:</h3>
+
+                <div class="modal-section-content">
+
+                  <section class="visibility-controls-primary">
```

Comment:
i think this should be a div, not a section:
> The section element represents a generic section of a document or application. A section, in this context, is a thematic grouping of content. The theme of each section should be identified, typically by including a heading (h1-h6 element) as a child of the section element.

> Examples of sections would be chapters, the various tabbed pages in a tabbed dialog box, or the numbered sections of a thesis. A Web site's home page could be split into sections for an introduction, news items, and contact information.

Source: [HTML5](http://www.w3.org/html/wg/drafts/html/CR/sections.html#the-section-element)